CODE/vulnscan.py

```
@@ -0,0 +1,186 @@
+    Args:
+        model_path_to_load (str): Path to the model file.
+
+    Returns:
+        model: Loaded model object.
+
+    Raises:
+        ValueError: If the model file format is unsupported.
+    """
+    if model_path_to_load.endswith('.pkl'):
+        return joblib.load(model_path_to_load)
+    elif model_path_to_load.endswith('.safetensors'):
```

Ruff (0.8.0)

Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)